src/tokens_entry.rs

```diff
@@ -612,28 +602,13 @@ impl<'a> TokensEntryBodyItem<&'a mut [u8]> {
         (self.iter_mut().ok()?).find(|entry| entry.id() == token_id)
     }
 
-    #[pre(
-        "Caller already increased the group size by `size_of::<TOKEN_ENTRY>()`"
-    )]
```

Comment:
Please make this a "/// Precondition:" comment or something.

Reply:
Done.